Makefile

```diff
@@ -53,6 +53,7 @@ cmd/util/util:
 .PHONY: unittest-main
 unittest-main:
 	# test all packages with Relic library enabled
+	cd insecure
```

Comment:
Temporary - just for debugging

Reply:
Removed in https://github.com/onflow/flow-go/pull/2936/commits/13e1b6bd5be297254e8be6e70c949f157191e6af